### PR TITLE
Moving linux vm builds to Ubuntu.

### PR DESCRIPTION
**Pull request by yusuf-goog**
Moving tests that have successfully run on Ubuntu machines in staging.

Bug:102406

*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ platform_properties:
         [
           {"dependency": "curl"}
         ]
-      os: Debian
+      os: Ubuntu
```
Comment (keyonghan) [inline .ci.yaml:28]: With platform value, we do not need target ones.

Comment (yusuf-goog) [reply .ci.yaml:28]: Removed them. thanks.

**Review by keyonghan — APPROVED** [review]
LGTM

**Comment by yjbanov** [comment]
I think the new HTML goldens look acceptable. For some reason in the past the HTML renderer used to render actual CJK characters, while all other platforms would render placeholders. The new golden is now showing placeholders for the HTML renderer, consistent with other platforms.

**Pull request merged**